pkg/api/types.go

```diff
@@ -403,6 +407,46 @@ const (
 	ClaimBound PersistentVolumeClaimPhase = "Bound"
 )
 
+// PersistentVolumeConditionType is a valid value for PersistentVolumeCondition.Type
+type PersistentVolumeConditionType string
+
+// These are valid conditions of PersistentVolume.
+const (
+	// PersistentVolumeBound means the volume is bound to a claim for storage and can be
+	// consumed as a storage resource by the claimant
+	PersistentVolumeBound PersistentVolumeConditionType = "Bound"
+	// PersistentVolumeRecycled is the recycled state of a PersistentVolume per the volume's reclaim policy
+	PersistentVolumeRecycled PersistentVolumeConditionType = "Recycled"
```

Comment:
To better fit the API construct it works with, perhaps `Reclaimed` would be more accurate.  This is the Condition of the volume's ReclaimPolicy (Retain/Recycle/Delete).

I'm thinking of "recycled" now as a discarded plastic water bottle in the blue bin.  What happens to it (via the ReclaimPolicy) at the plant is not my concern. I've thrown away my claim to that bottle.